addons/website_slides/static/src/xml/website_slides.xml

```diff
@@ -15,7 +15,11 @@
                         <form class="clearfix form-horizontal">
                             <div class="form-group">
                                 <div class="col-sm-4">
-                                    <img src="/website_slides/static/src/img/document.png" id="slide-image" title="Content Preview" class="img-responsive img-thumbnail"/>
+                                    <div class="img-thumbnail">
```

Comment:
I tried, but din't get that lucky ;) This particular fix is for the first point of DSU's review on 29th June 2015(from Archive Reviews)